src/components/Footer.jsx

```diff
@@ -70,7 +67,7 @@ const Footer = () =>
             <a href='/api'>API</a>
           </li>
           <li>
-            <GitHubCommitLink hash={GIT_COMMIT_HASH} short={GIT_SHORT_COMMIT_HASH} />
+            Latest Commit <GitHubCommitLink hash={GIT_COMMIT_HASH} short={GIT_SHORT_COMMIT_HASH} />
```

Comment:
True, I had someone in chat ask me what that random string was the other day because they thought I accidentally added it so I though I would add a label

Reply:
Ah I see. Might be able to just say "Version" or something 🤔